3.9-minimal/test/run

```diff
@@ -215,8 +215,7 @@ test_from_dockerfile(){
   info "Test from Dockerfile"
 
   # uwsgi in uwsgi-test-app
-  sed "s@#IMAGE_NAME#@${IMAGE_NAME}@" $test_dir/from-dockerfile/uwsgi.Dockerfile.tpl > $test_dir/from-dockerfile/Dockerfile
-  ct_test_app_dockerfile $test_dir/from-dockerfile/Dockerfile $test_dir/uwsgi-test-app 'Hello World from uWSGI hosted WSGI application!' app-src
+  ct_test_app_dockerfile $test_dir/from-dockerfile/uwsgi.Dockerfile.tpl $test_dir/uwsgi-test-app 'Hello World from uWSGI hosted WSGI application!' app-src
```

Comment:
Why is this `sed` removed? I don't see where it's replaced by anything else.

Reply:
Take a look here, it's already implemented as a part of the function: https://github.com/sclorg/container-common-scripts/blob/47b8b9cfe459a1bba3b04c44cc7da8519a3c520a/test-lib.sh#L863

Reply:
My plan is to change the function to sed the image name to the Dockerfile only if there is a placeholder for that and do nothing otherwise. That way, you can say whether the Dockerfile is ready for use or it needs the name of the image currently being tested.

But such change needs to check all the images where this function is already in use.

Reply:
Ah makes sense, and I agree that would be a good addition for the future.